pkgs/by-name/tk/tkgl/package.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  tcl,
+  fetchFromGitHub,
+  autoreconfHook,
+  tk,
+  libGL,
+}:
+
+tcl.mkTclDerivation {
+  pname = "tkgl";
+  version = "1.2.1-unstable-2025-06-01";
+
+  src = fetchFromGitHub {
+    owner = "3-manifolds";
+    repo = "TkGL";
+    rev = "45bf16e6c28a070c70fc9a0eb8c47a0b6ff8a2e3";
+    hash = "sha256-AA5LZGhMTWmTZqI/wtycUYLsUe9BwO3voGMS7vGlCM0=";
+  };
+
+  postPatch = ''
+    substituteInPlace configure.ac \
+      --replace-fail "-arch x86_64 -arch arm64" ""
+  '';
+
```

```suggestion
  postPatch = lib.optionalString stdenv.hostPlatform.isDarwin ''
    substituteInPlace configure.ac \
      --replace-fail "-arch x86_64 -arch arm64" ""
  '';
```
Am I right that this is only required for the build to run under Darwin?

---

Yes.